src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    @Override
+    public List<Machine> get() {
```

Comment:
The return type of `get()` should be `List<Excavator>` instead of `List<Machine>`. This ensures type safety and adheres to the checklist item: 'Do not use raw types. Always parametrize interfaces and classes if it is possible.'